packages/image/src/modifiers/removeImage.ts

```diff
@@ -0,0 +1,47 @@
+import { EditorState, Modifier, SelectionState } from 'draft-js';
+
+export default (editorState: EditorState, blockKey: string): EditorState => {
+  let content = editorState.getCurrentContent();
+  const newSelection = new SelectionState({
+    anchorKey: blockKey,
+    anchorOffset: 0,
+    focusKey: blockKey,
+    focusOffset: 0,
+  });
+
+  const afterKey = content.getKeyAfter(blockKey);
+  const afterBlock = content.getBlockForKey(afterKey);
+  let targetRange;
+
+  // Only if the following block the last with no text then the whole block
```

Comment:
Ta faltando pelo menos um "is" antes de "the last", mas fiquei confuso com a frase "Só se o bloco seguinte for o único sem texto" de qualquer forma, quando que o bloco de imagem teria texto?

Reply:
Para contextualizar: é o mesmo comportamento da remoção do Katex. No caso se trata do próximo bloco, não do bloco da imagem em si, está verificando se o próximo bloco/parágrafo tem conteúdo ou não.